packages/ui/src/views/connectors/connector-entity-form.tsx

```diff
@@ -162,14 +162,16 @@ export const ConnectorEntityForm: FC<ConnectorEntityFormProps> = ({
                 </Alert.Root>
               )}
             </EntityFormLayout.Form>
-          </Inner>
+          </Body>
           <Footer>
-            {isCreate && (
-              <Button variant="outline" onClick={() => onBack?.()}>
-                Back
-              </Button>
-            )}
-            <Button onClick={() => rootForm.submitForm()}>{isCreate ? 'Submit' : 'Apply changes'}</Button>
+            <ButtonGroup>
+              {isCreate && (
+                <Button variant="outline" onClick={() => onBack?.()}>
+                  Back
+                </Button>
+              )}
```

Comment:
Probably not much use in showing a back button if it won't do anything.
```suggestion
              {isCreate && onBack && (
                <Button variant="outline" onClick={onBack}>
                  Back
                </Button>
              )}
```